apps/prairielearn/elements/pl-variable-output/pl-variable-output.py

```diff
@@ -91,7 +91,7 @@ def render(element_html: str, data: pl.QuestionData) -> str:
     r_data = ""
     sympy_data = "from sympy import *\n\n"
     for child in element:
-        if child.tag == "variable":
+        if child.tag in {"variable", "pl-variable"}:
```

Comment:
Maybe add a comment that `variable` is accepted for backwards compatibility?

Reply:
Done here https://github.com/PrairieLearn/PrairieLearn/pull/7916/commits/ee958df312e9267177bbaa86510c2060a1816cc3